### PR TITLE
Make checkboxes and radios accessible/tabbable

### LINKED ISSUE #275: Bug: A11Y Problem with Checkbox Styling
When tabbing through a form, checkboxes (and presumably radios) don't tab. This is due to them having **[display](https://github.com/punchcard-cms/punchcard/blob/57f173f9c4476b2c0a13f7a21439f3020d950634/src/sass/partials/global/_mixins.scss#L54-L55): [none](https://github.com/punchcard-cms/punchcard/blob/57f173f9c4476b2c0a13f7a21439f3020d950634/src/sass/partials/global/_mixins.scss#L130-L131)** set. These need to be visually hidden, not removed entirely.

## Steps for Reproducing

- Go to a content type with checkboxes
- Try and tab to a checkbox
- Notice it skipped

## Screenshots

Can't Tab to Checkboxes

## Affected Browsers

- [ ] Chrome
- [ ] Firefox

(presumably all others)

## Runtime Version

v5.11

## Code Version

Latest

### PR DESCRIPTION
**Pull request by poofichu**
Resolves #275

### CODE REVIEW

**Comment by scottnath** [comment]
👍 
defer to @Snugug

**Comment by Snugug** [comment]
@poofichu Your branch needs to be updated

File: src/sass/partials/global/_mixins.scss [hunk 1]
```diff
@@ -62,11 +62,11 @@
       border: setting-get('input border width') solid setting-get('primary color');
       content: '';
       display: inline-block;
-      height: 1em;
-      margin-right: setting-get('rhythm') / 2;
-      transform: translateY(-.125em);
+      height: 1.25em;
+      margin-right: setting-get('rhythm') / 4;
+      transform: translateX(-.8rem);
```
Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: * You're mixing `rem` and `em` for translation in `checkbox`. You should use one or the other
* Where does the `.8` come from? That seems like a slightly arbitrary number

File: src/sass/partials/global/_mixins.scss [hunk 2]
```diff
@@ -78,6 +78,17 @@
...
     }
   }
+
+  &:focus {
+    outline: 0 none;
+
+    + label {
+
+      &::before {
+        outline: 4px solid setting-get('primary accent color');
```
Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: Is this the same outline we have throughout Punchcard? If so, we should probably turn it in to a variable to better reuse it

File: src/sass/partials/global/_mixins.scss [hunk 3]
```diff
@@ -139,17 +150,27 @@
       border-radius: 50%;
       content: '';
       display: inline-block;
-      height: 1em;
-      margin-right: setting-get('rhythm') / 2;
-      transform: translateY(-.125em);
+      height: 1.25em;
+      transform: translateX(-.9rem) translateY(-.125em);
```
Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: Again mixing `em` and `rem` for translate

File: src/sass/partials/global/_mixins.scss [hunk 4]
```diff
@@ -139,17 +150,27 @@
...
       vertical-align: middle;
-      width: 1em;
+      width: 1.25em;
     }
   }
 
   &:checked + label {
     &::before {
-      background: radial-gradient(color('grey', 80) 40%, transparent 40%);
+      background: radial-gradient(setting-get('primary color') 50%, setting-get('white') 50%);
```
Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: Where is this mixin coming from?

File: src/sass/partials/global/_mixins.scss (hunk 1 above)
Comment (poofichu) [inline src/sass/partials/global/_mixins.scss]: I was using `rem` to match up with what we're doing with txt/font but I can switch it to `em`

Comment (poofichu) [inline src/sass/partials/global/_mixins.scss]: Yes, .8 is arbitrary.  I'll find something that works with our `rhythm` setting as is divisable by all the things.

File: src/sass/partials/global/_mixins.scss (hunk 4 above)
Comment (poofichu) [inline src/sass/partials/global/_mixins.scss]: Grabbed the code from our original design guide.  Looks like it's coming from a node module.

Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: Can you please just write the standard Radial Gradient? That way we can can have [autoprefixer](https://github.com/postcss/autoprefixer) do its job

**Comment by Snugug** [comment]
w/r/t your emoji usage, as this was resolving a bug, it's not new functionality (:new: `:new:`), it's a bug fix (:bug: `:bug:`)

**Comment by poofichu** [comment]
Noted about the bug emoji.  Habit of doing 🆕

File: src/sass/partials/global/_mixins.scss (hunk 4 above)
Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: Oh, wow, NEVER MIND

Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: There's no `@include`. This is fine

**Comment by poofichu** [comment]
Even a @Snugug can make a mistake every now and then 😛

Now MERGE PLZZZZZZZZZ

File: src/sass/partials/global/_mixins.scss [hunk 5]
```diff
@@ -52,7 +52,7 @@
 }
 
 @mixin checkbox {
-  display: none;
+  transform: translateX(.5em);
```
Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: This doesn't entirely hide the radio buttons. I tried with `translateX(.25em) translateY(-.15em)` and that put the radio element entirely behind the custom styling. With `.5em` it peeks out the bottom right, which is very prominent on iOS

**.5em**
screen shot 2016-09-01 at 1 48 08 pm
screen shot 2016-09-01 at 1 48 03 pm

**.25em .15em**
screen shot 2016-09-01 at 1 48 31 pm
screen shot 2016-09-01 at 1 48 15 pm

File: src/sass/partials/global/_mixins.scss [hunk 6]
```diff
@@ -128,7 +138,7 @@
 }
 
 @mixin radio {
-  display: none;
+  transform: translateX(.5em);
```
Comment (Snugug) [inline src/sass/partials/global/_mixins.scss]: This doesn't entirely hide the radio buttons. I tried with `translateX(.25em) translateY(-.15em)` and that put the radio element entirely behind the custom styling. With `.5em` it peeks out the bottom right, which is very prominent on iOS

**.5em**
screen shot 2016-09-01 at 1 48 08 pm
screen shot 2016-09-01 at 1 48 03 pm

**.25em .15em**
screen shot 2016-09-01 at 1 48 31 pm
screen shot 2016-09-01 at 1 48 15 pm

**Comment by Snugug** [comment]
@poofichu Out of date again

**Pull request merged**